Update docs to highlight need for git-submodule when using docker (#7281)
qmk/qmk_firmware: docs/getting_started_build_tools.md

## Docker

If this is a bit complex for you, Docker might be the turnkey solution you need. After installing [Docker CE](https://docs.docker.com/install/#supported-platforms), run the following commands from the `qmk_firmware` directory to build a keyboard/keymap:
```bash

> # If this is the first time, ensure your submodules (third-party libraries) are up to date by running:
> util/docker_build.sh git-submodule

Reviewer: ```suggestion
# If this is the first time, or it has been a while since you've updated, ensure your submodules (third-party libraries) are up to date by running:
make git-submodule
```
